build.gradle

```diff
@@ -14,18 +14,16 @@ buildscript {
     repositories {
         google()
         maven { url 'https://plugins.gradle.org/m2/' }
-        jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.3'
+        classpath 'com.android.tools.build:gradle:4.2.0'
```

Comment:
Updating to AGP 4.2.0 requires minimum Gradle version 6.7.1

Reply:
Ooh god, I can see that now, wasn't aware that the library had a `gradle-wrapper.properties` as well... -.-'

Should be fixed now!